tools/ci/tc/tests/test_valid.py

```diff
@@ -145,7 +167,6 @@ def test_verify_payload():
       'wpt-firefox-nightly-stability',
       'wpt-firefox-nightly-results',
       'wpt-firefox-nightly-results-without-changes',
-      'wpt-chrome-dev-stability',
```

Comment:
I'm curious as to why this did not cause a failed test until I made recent changes... The new `filter_excluded_users` function should cause a mismatch here, which would always fail this using the original test data.

[This test started to correctly fail as expected after this change.](https://github.com/web-platform-tests/wpt/pull/31966/commits/6a34484a775926a7adcf699210e908571cf445d7)

It worries me that maybe my old implementation did not work/exclude as intended, but I cannot find the error in it besides being more inefficient.